Model/Indexer/Data.php

```diff
@@ -42,30 +42,38 @@
 use Nosto\Tagging\Model\ResourceModel\Product\Index\Collection as IndexCollection;
 use Nosto\Tagging\Model\ResourceModel\Product\Index\CollectionFactory as IndexCollectionFactory;
 use Nosto\Tagging\Model\Service\Index as NostoIndexService;
+use Nosto\Tagging\Helper\Account as NostoHelperAccount;
+use Magento\Store\Model\Store;
 
 /**
  * An indexer for Nosto product sync
  */
 class Data implements IndexerActionInterface, MviewActionInterface
 {
-    public const INDEXER_ID = 'nosto_product_index_data';
+    public const INDEXER_ID = 'nosto_index_product_data_sync';
```

Comment:
Will there be any issues with the upgrade?

Reply:
No.